### PR TITLE
Profile input field position fix - Android

### LINKED ISSUE #16825: "Your name" input field is dispositioned to the bottom on Create profile screen (Android)
Reproduced on Android 12, Samsung Galaxy A52; 
Pixel 7a

#### Expected behavior

Input field should be centered 

[Figma](https://www.figma.com/file/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?type=design&node-id=2241-250700&mode=design&t=B85RWSh5foecoM3z-0)

Onboarding for Mobile – Figma 2023-08-01 11-50-17

#### Actual behavior

Input field is dispositioned to the bottom

photo_2023-08-01 11 51 07

### Reproduction

1. Proceed to Create profile screen
2. Pay attention at the input field position

### Additional Information

- Status version: nightly 
- Operating System: Android

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by ajayesivan**
fixes #16825 

### Summary
Profile input field - input position fix for Android.

#### Platforms

- Android

#### Areas that maybe impacted
Screens where we are using the Profile Input component
- Quo2 Preview -> Profile Input
- Create Profile Screen -> Your Name input

status: ready

### CODE REVIEW

File: src/quo2/components/inputs/profile_input/style.cljs [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
-  (:require [react-native.platform :as platform]
-            [quo2.foundations.colors :as colors]))
+  (:require [quo2.foundations.colors :as colors]))
 
 (defn container
   [customization-color]
   {:background-color   (colors/custom-color customization-color 50 40)
    :padding-horizontal 12
    :padding-top        12
-   :padding-bottom     (if platform/ios? 10 0)
```
Comment (ajayesivan) [inline src/quo2/components/inputs/profile_input/style.cljs:10]: Hello @alwx, I've removed this condition you added via #16923 in order to resolve the Android issue. Following this change, I couldn't detect any issues, but please let me know if I've overlooked something.

**Review by briansztamfater — APPROVED** [review]
LGTM!

**Review by mohsen-ghafouri — APPROVED** [review]

**Review by BalogunofAfrica — APPROVED** [review]

**Comment by qoqobolo** [comment]
Thanks for the fix @ajayesivan!
PR is ready to be merged.

File: src/quo2/components/inputs/profile_input/style.cljs (hunk 1 above)
Comment (alwx) [reply src/quo2/components/inputs/profile_input/style.cljs:10]: Does it still match the design?

**Review by alwx — CHANGES_REQUESTED** [review]
Asked a question in code

**Review by alwx — APPROVED** [review]

File: src/quo2/components/inputs/profile_input/style.cljs (hunk 1 above)
Comment (ajayesivan) [reply src/quo2/components/inputs/profile_input/style.cljs:10]: Yes.

**Comment by ajayesivan** [comment]
@qoqobolo, considering that this PR involves UI changes, should we also request a design review?

**Comment by qoqobolo** [comment]
@ajayesivan these changes are quite small, I think that the design review can be skipped this time so as not to delay the release branch cut.
If any UI issues are noticed, we will report them later.

**Pull request merged**